spim_registration/timelapse/cluster.json

```diff
@@ -7,12 +7,12 @@
 
     "resave_hdf5" :	
     {
- 	"lsf_extra" : "-n 7 -R \"span[hosts=1] rusage[mem=50000]\""
+ 	"lsf_extra" : "-n 2 -R \"span[hosts=1] rusage[mem=50000]\""
```

Comment:
The reason why we increased the number of cores is not that the process necessarily needs that many resources. 
The problem was that the threads expanded to other cores. By increasing the number of cores reserved to these jobs this problem was partly compensated. Now with sysconfcpus the jobs seem to be contained within the assigned number of cores and this is not necessary anymore. The settings in the cluster.json now reflect the settings of sysconfcpus. In fact it would be great if sysconfcpus would be determined by the cluster.json.